### DISCUSSION TITLE
Missing font & image

### DISCUSSION THREAD

**Discussion by M35S** [ldicker83/Micropolis-SDLPP#14] [Q&A]
I managed to build on Visual Studio 2019 C++ 17 but upon debugging found there are currently two missing assets:

Raleway-Medium.ttf 
ToolPalette.png

Had a look online for alternatives and found a font but without knowing what ToolPalette.png actually looks like I cannot progress further :( 

Anyone able to point me to the right direction for Micropolis assets that would match exactly with this project? Thanks.

**Comment by ldicker83** [comment]
![ToolPalette](https://github.com/user-attachments/assets/8bcb1a7f-2b5c-45c7-885a-8ec88b187dee)

Hello! Apologies for taking so long to get back to you. I attached the file here but I'm going to commit to the repository so you can get it that way as well.

Sorry about that!
